.github/workflows/ci.yaml

```diff
@@ -46,19 +46,29 @@ jobs:
       COVERAGE_RCFILE: ${{ github.workspace }}/.coveragerc
     steps:
     - uses: actions/checkout@v4
-    - uses: conda-incubator/setup-miniconda@v3
+
+    - name: Set cache key
+      id: cache-key
```

Comment:
This `id:` isn't needed since the step is never referred to anywhere…

Reply:
Removed in abe4b8bf82bcd0dc23743dff0d5e5a749121f40b